Properties/launchSettings.json

```diff
@@ -18,7 +18,12 @@
     "ReleaseNotes_WebAPI": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "HOST": "localhost",
+        "PORT": "5432",
+        "DB_NAME": "releasenotedb",
+        "DB_USER": "dbuser",
+        "DB_PASSW": "dbpassword"
```

Comment:
Koffer vil du ha inn det her i repositoriet?

Reply:
Beklager, du har skrevet koffer

Reply:
Kan vi ikke bære ignore den fila?

Reply:
Eventuelt så kan du lag egne launchSettings i din lokale Jetbrains Rider.
![image](https://user-images.githubusercontent.com/33400600/79247505-ed442f00-7e7a-11ea-829c-78125843a744.png)

Reply:
1. Launchsettings

> 
> 
> Eventuelt så kan du lag egne launchSettings i din lokale Jetbrains Rider.
> ![image](https://user-images.githubusercontent.com/33400600/79247505-ed442f00-7e7a-11ea-829c-78125843a744.png)

envs du edita der blir lagra til launchSetting.json

Reply:
Uff

Reply:
> 
> 
> Kan vi ikke bære ignore den fila?

Bør ikkje det, sia fila e krevd for å kjøre prosjektet. Treng ikkje å gjer det meir vanskelig enn nødvendig å klone repoet. I tillegg e dissa parametera kun for development mode, så der e ingen secrets.

Reply:
Det e sant, helt uinterresant om den påloggningsinformasjonen bi funnet av nån andre.

Reply:
Men e det ikke på tide å få dockern tell å fungere? Det ska jo ikke vær umulig

Reply:
Livskvaliteten min he økt sia den slutta å fungere

Reply:
Buildtimes på serveren ned fra >30 sekund til 7 sekund

Reply:
Det mest interresante e jo koffer det e så tregt. E det bære pc'n din som e treg, e det Windows eller e det nå som e feil?

Reply:
Okei, eg e overbevist. Kjør på!